### PR TITLE
Customisable foreground + background colours

### PR DESCRIPTION
**Pull request by taxomania**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -17,9 +17,9 @@ allprojects {
 
 subprojects {
     group = 'com.justyoyo.contrast'
-    version = '0.1.0-SNAPSHOT'
+    version = '0.1.1-SNAPSHOT'
 }
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '1.12'
+    gradleVersion = '2.3'
```
Comment (chrisjenx) [inline build.gradle:24]: Your gradle wrapper version does not match your bin. Don't foget to run `gradle wrapper` after updating the task.

**Comment by alistair-broomhead** [comment]
Why did this appear in my notification area? Oh and hi guys!

**Comment by chrisjenx** [comment]
I'm guessing because it an opensource repo, and we would of been subscribed to notifications :)

File: build.gradle (hunk 1 above)
Comment (taxomania) [inline build.gradle:24]: woops, good spot!

**Pull request merged**